src/color/p5.Color.js

```diff
@@ -290,6 +294,34 @@ class Color {
     }
     return colorString;
   }
+/**
+   * Checks if two colors contrast ratio is WCAG 2.1 compliant and returns the ratio
+   *
+   * @param {Color} other
+   * @returns {{ ratio: Number, passes: boolean }}
```

Comment:
Do you know if `passes` is using AA or AAA or something else?

Reply:
Hi both, I added some notes about this in this comment: https://github.com/processing/p5.js/pull/7944#issuecomment-3073071849 The intent is to return the ratio and provide 1-2 threshholds constants as global vars to avoid recommending hardcoded numbers when this is used